src/lib/middleware/proxyReflection.ts

```diff
@@ -1,50 +0,0 @@
-import type { NextFunction } from "express"
-import type { ArtsyRequest, ArtsyResponse } from "./artsyExpress"
-
-import httpProxy from "http-proxy"
```

Comment:
I hoped to drop some of these dependencies, but it looks like they're all used elsewhere.